src/test/ui/nll/closure-requirements/region-lbr-named-does-not-outlive-static.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// Basic test for free regions in the NLL code. This test ought to
+// report an error due to a reborrowing constraint. Right now, we get
+// a variety of errors from the older, AST-based machinery (notably
+// borrowck), and then we get the NLL error at the end.
+
+// compile-flags:-Znll -Zborrowck=mir
```

Comment:
Why do you have different tests with `-Z nll` and with `-Z nll -Z borrowck=mir`?

Reply:
Mistake. It used to be that `-Znll` implies `-Zborrowck-mir`  so maybe it dates from then. They should really all have `-Znll -Zborrowck=mir -Zverbose`.